lib/matplotlib/backends/backend_gtk.py

```diff
@@ -1003,13 +999,13 @@ def on_combobox_lineprops_changed(self, item):
         if marker is None: marker = 'None'
         self.cbox_markers.set_active(self.markerd[marker])
 
-        r,g,b = colorConverter.to_rgb(line.get_color())
-        color = gtk.gdk.Color(*[int(val*65535) for val in (r,g,b)])
+        rgba = mcolors.to_rgba(line.get_color())
```

Comment:
The example runs with py2/gtk2.  However, it is the typical example of why I want to get rid of `to_rgb`: the line props dialog seemingly support setting alpha (the glade file explicitly sets `use_alpha` to True), but the alpha is silently dropped after.